ui/common/src/socket.ts

```diff
@@ -41,47 +41,50 @@ interface Settings {
   params?: Partial<Params>;
   options?: Partial<Options>;
 }
-
-const origSend = WebSocket.prototype.send;
```

Comment:
remove unnecessary `WebSocket.prototype` access

Reply:
it's useful against some cheat bots that attempt to override the websocket send method.